lib/routes/charge/items/catalog_item.dart

```diff
@@ -83,12 +89,38 @@ class CatalogItem extends StatelessWidget {
     ]);
   }
 
-  String _formattedPrice() {
-    var formatter = CurrencyWrapper.fromShortName(_itemInfo.currency, accountModel);
-    return formatter.format(_itemInfo.price, includeCurencySuffix: true, removeTrailingZeros: true);
+  InlineSpan _buildCurrencySymbol() {
+    return _itemInfo.currency == "SAT"
+        ? WidgetSpan(
+            child: Container(
+              height: 20,
+              child: Align(
+                child: Text(_getSymbol(), // Icon
+                    style: TextStyle(fontSize: 13.4, fontFamily: 'SAT')),
+                alignment: Alignment.center,
+              ),
+            ),
+          )
+        : TextSpan(
+            text: _getSymbol(), // Icon
+          );
+  }
+
+  String _getSymbol() {
+    var currency = Currency.fromTickerSymbol(_itemInfo.currency);
```

Comment:
Instead you can use the `CurrencyWrapper.fromShortName` to get the unified API for different currencies.
I makes sense also to add `symbol` property on the `CurrrencyWrapper`.

Reply:
[Done](https://github.com/breez/breezmobile/pull/292/commits/a5949de69578d9dcc7dac4b8ac73ce14e8784a80)